openlibrary/plugins/worksearch/subjects.py

```diff
@@ -47,6 +47,10 @@ class subjects(delegate.page):
     path = '(/subjects/[^/]+)'
 
     def GET(self, key):
+        from openlibrary.plugins.worksearch.code import search
+        s = key.split('/')[2]
+        return search().GET(subject=s)
+
         nkey = self.normalize_key(key)
```

Comment:
This introduces a lot of dead code; some of this also feels like it might still be necessary (like the key normalization). Can we put some more effort into making this a little more streamlined? Also, if we've deleted the old template, we can delete it now :)